app/views/layouts/govuk_admin_template.html.erb

```diff
@@ -94,6 +94,12 @@
     <% end %>
     <section class="<%= content_for?(:full_width) ? 'container-fluid' : 'container' %>">
       <main role="main">
+        <% if GovukAdminTemplate::Config.show_flash %>
+          <% flash.each do |type, message| %>
```

Comment:
This assumes that `flash` only contains messages, which isn't always the case:
https://github.com/alphagov/transition/blob/master/app/controllers/concerns/track_mappings_progress.rb#L23-L25